enhancements/installer/aws-outpost-remote-workers.md

```diff
@@ -0,0 +1,209 @@
+---
+title: aws-outpost-remote-workers
+authors:
+  - "@nirarg"
+  - "@pkliczewski"
+reviewers:
+  - "@sdodson"
+approvers:
+  - "@sdodson"
+api-approvers:
+  - None
+creation-date: 2022-10-11
+last-updated: 2022-10-11
+tracking-link:
+  - https://issues.redhat.com/browse/OCPPLAN-9617
+  - https://issues.redhat.com/browse/ECOPROJECT-866
+see-also: []
+replaces: []
+superseded-by: []
+---
+
+
+# Remote workers on AWS outpost
+
+## Summary
+
+Running workload closer to end users or leveraging data locality seems to be
+important for many customers. That is why AWS came up with fully [managed
+service](https://docs.aws.amazon.com/outposts/latest/userguide/how-outposts-works.html)
+that extends AWS infrastructure and run on customer's premises. This enhancement
+proposes extending how we deploy OCP on AWS platform so it ispossible to run on
+AWS Outpost.
+
+The work is divided into several phases:
+- Phase 0: Research. Understand what is needed to deploy OCP on AWS Outpost.
+  No code changes needed and we only provide documentation on how to customize
+  generated manifests and list known limitations.
+- Phase 1: Improve cluster deployment on AWS Ouptost by implementing necessary
```

Comment:
Which teams are committed to delivering on Phase 1 and 2 goals? And when?

Reply:
Phase 1 needs to be handled by corresponding teams within OCP unless we decide to keep the limitations. For Phase 2 is in progress and my team is working on it.